velox/exec/tests/utils/OperatorTestBase.h

```diff
@@ -118,6 +118,12 @@ class OperatorTestBase : public testing::Test,
     return test::assertQuery(plan, {expectedResults});
   }
 
+  std::shared_ptr<Task> assertQuery(
+      const CursorParameters& params,
+      const RowVectorPtr& expectedResults) {
+    return test::assertQuery(params, {expectedResults});
```

Comment:
This function just calls another one. Wondering if we can use 'test::assertQuery' directly.

Reply:
This function is similar to https://github.com/facebookincubator/velox/blob/main/velox/exec/tests/utils/OperatorTestBase.h#L115, but accept `CursorParameters`.

To align with usage, I add this function, the `assertQuery` should have a function with input argument planNode and corresponding params to set config.